stable/elastabot/values.yaml

```diff
@@ -21,6 +21,9 @@ elasticsearch:
 # Elastalert settings. Elastalert chart information is available here:
 # https://github.com/kubernetes/charts/tree/master/stable/elastalert
 elastalert:
+  # Index prefix for elastalert
+  index: "elastalert"
```

Comment:
```suggestion
  index: "elastalert_status"
```
Shouldn't the previous default value be kept? Isn't it a breaking change otherwise?